servicex/resources/submit_transformation_request.py

```diff
@@ -186,8 +187,15 @@ def post(self):
                     )
                 }
 
+                parsed_did = DIDParser(requested_did,
+                                       default_scheme=current_app.config[
+                                           'DID_FINDER_DEFAULT_SCHEME'])
```

Comment:
```suggestion
                parsed_did = DIDParser(requested_did,
                                       default_scheme=current_app.config.get(
                                           'DID_FINDER_DEFAULT_SCHEME', "rucio"))
```
Use `dict.get` in case the config value is missing (minor downside - rucio default is hardcoded).